RELEASE_INFO-6.7.md

```diff
@@ -8,13 +8,18 @@
 
 ### Removal of old `changelog` handling
 As we changed how we process and generate changelogs the "old" changelog files are no longer needed.
-Therefore, we removed all the internal code used to generate and validate them. 
+Therefore, we removed all the internal code used to generate and validate them.
 The whole `Shopware\Core\Framework\Changelog` namespace was removed. The code is not needed anymore, you should adjust the `RELEASE_INFO` and `UPGRADE` files manually instead.
 
 ## Administration
 
 ## Storefront
 
+### Language selector now displays language with territory
+When the `v6.8.0.0` feature flag is active, the language selector in `src/Storefront/Resources/views/storefront/layout/header/actions/language-widget.html.twig` now displays language names with their territory in the dropdown (e.g., "Deutsch (Deutschland)", "English (United Kingdom)") instead of just the language name.
```

Comment:
I would not speak of the feature flag, but more like that: "with the next major version..."
"the css flags are deprecated and will be removed with..." is also missing.